### PR TITLE
[2] - Elasticsearch and Kibana added to Docker

### PR DESCRIPTION
**Pull request by wricka**

### CODE REVIEW

File: docker-compose-prod.yml [hunk 1]
```diff
@@ -62,3 +62,20 @@ services:
         - "15672:15672"
       volumes:
         - ./var/rabbitmq:/var/lib/rabbitmq
+
+    elasticsearch:
+      image: docker.elastic.co/elasticsearch/elasticsearch:7.5.1
```
Comment (LeWricka) [inline docker-compose-prod.yml:67]: [Testing] - Testing comment

File: docker-compose.yml [hunk 2]
```diff
@@ -68,3 +68,102 @@ services:
         - "15672:15672"
       volumes:
         - ./var/rabbitmq:/var/lib/rabbitmq
+
+    elasticsearch:
```
Comment (LeWricka) [inline docker-compose.yml:72]: [Naming] - Naming comment

**Review by LeWricka — CHANGES_REQUESTED** [review]

File: docker-compose.yml [hunk 3]
```diff
@@ -68,3 +68,102 @@ services:
...
+      image: docker.elastic.co/elasticsearch/elasticsearch:7.5.1
+      container_name: elasticsearch
+      environment:
+        - node.name=elasticsearch1
```
Comment (wricka) [inline docker-compose.yml:76]: [Architecture]: Is this correct?

File: src/Infrastructure/Controller/AddCommentController.php [hunk 4]
```diff
@@ -2,13 +2,20 @@
...
 
 class AddCommentController
 {
-    public function __invoke()
+    public function __invoke(LoggerInterface $logger)
     {
         echo 'hello Iker';
+        $logger->info('I just got the logger !!!!!!!');
+        $logger->info('I just got the logger !!!!!!!');
+        $request = Request::createFromGlobals();
```
Comment (wricka) [inline src/Infrastructure/Controller/AddCommentController.php:16]: [Testing]: Another testing comment

File: docker-compose-prod.yml (hunk 1 above)
Comment (wricka) [reply docker-compose-prod.yml:67]: ok

**Review by LeWricka — APPROVED** [review]
Ok to merge

**Pull request merged**